Algebras/MFE/alg_mfe.gap

```diff
@@ -17,6 +17,47 @@ algebra alg_mfe implements sig_foldrna(alphabet = char, answer = int) {
   int cadd_Pr_Pr(int le,int re) {return le+re;}
   int cadd_Pr(int le,int re) {return le+re;}
 
+  int sadd_cut(Subsequence lb, Subsequence c, int x) {
```

Comment:
similarly to dotBracket, we might want to move those new parts into its only algpart file